src/XRP/xrp-transaction.ts

```diff
@@ -0,0 +1,109 @@
+import { Transaction } from '../generated/web/org/xrpl/rpc/v1/transaction_pb'
+import RippledFlags from '../rippled-flags'
+import XRPSigner from './xrp-signer'
+import XRPTransactionType from './xrp-transaction-type'
+import XRPPayment from './xrp-payment'
+import XRPMemo from './xrp-memo'
+
+/* A transaction on the XRP Ledger.
+ *
+ * @see: https://xrpl.org/transaction-formats.html
+ */
+// TODO(amiecorso): Modify this object to use X-Address format.
+export default class XRPTransaction {
+  public static from(transaction: Transaction): XRPTransaction | undefined {
+    const account = transaction
+      .getAccount()
+      ?.getValue()
+      ?.getAddress()
+    const fee = transaction.getFee()?.getDrops()
+    const sequence = transaction.getSequence()?.getValue()
+    const signingPublicKey = transaction.getSigningPublicKey()?.getValue()
+    const transactionSignature = transaction
+      .getTransactionSignature()
+      ?.getValue()
```

Comment:
use `getValue_u8()`

Reply:
I did this in two other places that previously had polymorphic fields - now it's all byte arrays